spacy_llm/tests/tasks/test_spancat.py

```diff
@@ -0,0 +1,464 @@
+# mypy: ignore-errors
```

Comment:
Did you mean to leave this long-term?

Reply:
Ideally we don't ignore errors file-wide.

Reply:
I copy-pasted the test suite for the NER. I'll change that in both files.